torchgen/api/python.py

```diff
@@ -1163,6 +1163,8 @@ def dispatch_lambda_arg(cpp_arg: Binding) -> DispatchLambdaArgument:
     "::std::tuple<double,int64_t>",
     "::std::tuple<at::Tensor,::std::vector<at::Tensor>>",
     "::std::vector<at::Tensor>",
+    # for cslt_sparse_mm_search
+    "::std::tuple<int64_t,int64_t,bool,int64_t>",
```

Comment:
There's a warning above that says

```
# [old codegen] XXX: if you got here because of an assertion failure, it doesn't mean
# it's enough to just extend the list here. Before you do this, make sure
# to add an appropriate wrap() overload in torch/csrc/autograd/utils/wrap_outputs.h.
```

Is that still relevant here?

Reply:
wrap() is for converting the the C++ dtypes to the python dtypes (int64_t -> python int), we don't need to add a new one here since tuple, int and bool are all already implemented.